infinispan-sample-app/pom.xml

```diff
@@ -1,51 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
```

Comment:
Could you please create a sample app in [the tutorials](https://github.com/infinispan/infinispan-simple-tutorials). Of course it won't compile now but once we release the starters it should work fine.